coralproject/talk
# Pw reset logs

## What does this PR do?

Ok, so somehow the full sendgrid connection string is trying to use http over https or vice versa. There's no way to pass a full connection string with api key and disable secure connections as far as I can tell. So, I just switched over the config to use the send grid user/pass instead and added a link to the docs so future implementers can easily plugin whatever service they use.

## How do I test this PR?

- set the env vars `TALK_SMTP_USERNAME` and `TALK_SMTP_PASSWORD`
- restart the app
- try to reset your password
- you should receive an email
